packages/@tailwindcss-node/src/compile.ts

```diff
@@ -100,11 +100,8 @@ async function importModule(path: string): Promise<any> {
   try {
     return await import(path)
   } catch (error) {
-    try {
-      jiti ??= createJiti(import.meta.url, { moduleCache: false, fsCache: false })
-      return await jiti.import(path)
-    } catch {}
-    throw error
+    jiti ??= createJiti(import.meta.url, { moduleCache: false, fsCache: false })
+    return await jiti.import(path)
   }
```

Comment:
I had a typo in a `.ts` file that was loaded. The error message only showed that it could not load `.ts` file while I wanted to see that there was a parse error instead. Since jiti will also show parse errors for regular JS files, we can just rely on that error instead.